### ISSUE TITLE
bach_size设置问题

### ISSUE THREAD

**Issue by 1994su** [csuhan/ReDet#80]
请问一下咱们模型训练的batchsize在哪里修改？没有看到相关参数，目前用的是8卡 gpu，设置多少batchsize合适？

**Comment by csuhan** [comment]
Batch_size:
https://github.com/csuhan/ReDet/blob/0b9addf3c2734659fd6ffc7824f2e659fde4419c/configs/ReDet/ReDet_re50_refpn_1x_dota1.py#L142

Copied from https://github.com/csuhan/ReDet/blob/master/README.md
> All our models are trained on 4GPUs with a learning rate 0.01. If you train your model with more/fewer GPUs, remember to change the learning rate, e.g., 0.02lr=0.0025lr*8GPU, 0.0025lr=0.0025lr*1GPU.
